src/connection/postgres_connection_factory.py

```
@@ -0,0 +1,47 @@
+        data = None
+        
+        try:
+            with open(file_path) as file:
+                data = json.load(file)
+        except IOError as e:
+            raise IOError(f"Falha ao ler arquivo:\n{e}")
+
+        result = data[conn_id]
+        
+        if len(result) == 0:
+            raise KeyError(f"Nome da conexão não existe")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)